src/libs/actions/Report.js

```diff
@@ -495,6 +495,85 @@ function fetchChatReports() {
         });
 }
 
+/**
+ * Get a simplified version of an IOU report
+ *
+ * @param {Number} reportID
+ * @param {Object} reportData
+ * @param {Number} reportData.transactionID
+ * @param {Number} reportData.amount
+ * @param {String} reportData.currency
+ * @param {String} reportData.created
+ * @param {String} reportData.comment
+ * @param {Object[]} reportData.transactionList
+ * @param {String} reportData.ownerEmail
+ * @param {String} reportData.managerEmail
+ * @returns {Object}
+ */
+function getSimplifiedIOUReport(reportID, reportData) {
+    const transactions = _.map(reportData.transactionList, transaction => ({
+        transactionID: transaction.transactionID,
+        amount: transaction.amount,
+        currency: transaction.currency,
+        created: transaction.created,
+        comment: transaction.comment,
+    }));
+
+    return {
+        reportID,
+        ownerEmail: reportData.ownerEmail,
+        managerEmail: reportData.managerEmail,
+        currency: reportData.currency,
+        transactions,
+    };
+}
+
+/**
+ * Fetches the updated data for an IOU Report and updates the IOU collection in ONYX
+ *
+ * @param {Number} reportID
+ * @param {Object[]} reportHistory
+ */
+function updateIOUReportData(reportID, reportHistory) {
+    const containsIOUAction = _.any(reportHistory, action => action.actionName === 'IOU');
+
+    // If there aren't any IOU actions, we don't need to fetch any additional data
+    if (!containsIOUAction) {
+        return;
+    }
+
+    const otherParticipants = _.chain(reportHistory)
+        .pluck('actorEmail')
+        .unique()
+        .without(currentUserEmail)
+        .value();
+
+    // If we have more than one participant, this is not an IOU
+    if (otherParticipants.length > 1) {
```

Comment:
cc @Julesssss just double-checking that it isn't possible for an IOU to have more than 2 participants. The API command for `getIOUReport` only takes one debtorEmail atm

Reply:
That's right, each IOU report has a 1:1 relationship. Bill splits are a collection of 1:1 IOU Reports.